python/pyspark/sql/functions/builtin.py

```diff
@@ -14228,6 +14253,17 @@ def try_element_at(col: "ColumnOrName", extraction: "ColumnOrName") -> Column:
     +-----------------------+
     |                   NULL|
     +-----------------------+
+
+    Example 6: Getting a value from a map using a column name as the key
+
+    >>> from pyspark.sql import functions as sf
+    >>> df = spark.createDataFrame([({"a": 1.0, "b": 2.0}, "a")], ['data', 'b'])
+    >>> df.select(sf.try_element_at(df.data, 'b')).show()
```

Comment:
Do we need to add another example below this?
```
df.select(sf.try_element_at(df.data, df.b)).show()
```
Because of the grammar scene above, I took `a long time` to understand its intended meaning, which is so `obscure`, 😄